tests/distributed_grids/2d_coarse_grid_01.cc

```diff
@@ -67,7 +67,7 @@ void test(std::ostream & /*out*/)
 
 int main(int argc, char *argv[])
 {
-  Utilities::MPI::MPI_InitFinalize mpi_initialization(argc, argv, 1);
+  Utilities::MPI::MPI_InitFinalize mpi_initialization (argc, argv, 1);
```

Comment:
Formatting them all the same way made this (and future) regex replace a lot easier.